internal/adapters/entrypoints/rest/handlers/delete_trusted_account.go

```diff
@@ -0,0 +1,34 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/rsksmart/liquidity-provider-server/internal/adapters/entrypoints/rest"
+	lpuc "github.com/rsksmart/liquidity-provider-server/internal/usecases/liquidity_provider"
+	"github.com/rsksmart/liquidity-provider-server/pkg"
+)
+
+// NewDeleteTrustedAccountHandler
+// @Title Delete Trusted Account
+// @Description Deletes a trusted account
+// @Param TrustedAccountAddressRequest body pkg.TrustedAccountAddressRequest true "Address of the trusted account to delete"
+// @Success 204 object
+// @Route /management/trusted-accounts [delete]
+func NewDeleteTrustedAccountHandler(useCase *lpuc.DeleteTrustedAccountUseCase) http.HandlerFunc {
+	return func(w http.ResponseWriter, req *http.Request) {
+		var err error
+		request := &pkg.TrustedAccountAddressRequest{}
```

Comment:
Its better to avoid using the body in a DELETE request, https://developer.mozilla.org/en-US/docs/Web/HTTP/Reference/Methods/DELETE prefer using path variables in this case